src/ctx.rs

```diff
@@ -26,14 +26,24 @@ impl Ctx {
         #[cfg(feature = "native-tls")]
         {
             use std::sync::Arc;
-            let client = ureq::builder()
-                .tls_connector(Arc::new(native_tls_crate::TlsConnector::new()?))
-                .build();
-            Ok(client)
+            let mut builder =
+                ureq::builder().tls_connector(Arc::new(native_tls_crate::TlsConnector::new()?));
+            if let Ok(proxy) = std::env::var("https_proxy") {
```

Comment:
I never use HTTP proxies, is this the conventional way to specify them to programs that use them? I'm just thinking that usually I prefer environment variables be upper cased and prefixed for the application, so something like `XWIN_HTTPS_PROXY`, but if users are expecting `https_proxy` to work that's fine as well.

Reply:
No, this is fine if it's the conventional form since I guess you probably just set this globally and expect it work for the program that perform HTTP calls, I just wanted to make sure.